Update the modal text to explain a bit more about the app.

vituri opened the pull request:
*Have you read the [Contributing Guidelines](https://github.com/Appsilon/.github/blob/main/CONTRIBUTING.md)?*

## Description
Update the modal text to explain a bit more about the app.

I chose to let the "Powered by" section as-is, because the text fits the screen without problems and it looks nice.

![image](https://github.com/user-attachments/assets/4604b8b0-019d-483f-8e03-1d3a9064ed3a)

## Definition of Done
- [ ] The change is thoroughly documented.
- [ ] The CI passes (`R CMD check`, linter, unit tests, spelling).
- [ ] Any generated files have been updated (e.g. `.Rd` files with `roxygen2::roxygenise()`)

Commit:
remove first link to Appsilon; add small description

Commit:
chore: lint

Commit:
refactor: enhance about section description and formatting

jakubnowicki commented on app/view/about_section.R:
```diff
@@ -28,6 +28,31 @@ about_section <- shiny$div(
   )
 )
 
+about_section <- shiny$div(
```
1. Your commit message was about removing the link, but actually you are only overwriting the `about_section` object. If you want to remove something, just remove it.
2. But, the link should not be removed! It should be included in the section about Appsilon!!!

vituri replied:
I previously removed the link because we already have a "Let's talk" button and the clickable image on the brand section.

I included the link again, now inside `brand_section` at the end, as a "Learn more about Appsilon".

jakubnowicki commented on app/logic/connect-extension.R:
```diff
@@ -1,5 +1,5 @@
 box::use(
```
I missed this in the PR that added the manifest, but now I see that it exists. This file is not supposed to be here, as this is not a part of the application! This should be just a snippet in the documentation, not a file hidden in the application code!. Please remove it in a separate PR.

vituri replied:
I will remove this file for now and move it to a PR about documentation to developers: why this branch exists, how to update it, how to generate the files and so on.

jakubnowicki replied:
This is not what I suggested. Now, instead of a single PR, we will have two that include changes related to this..

jakubnowicki commented on app/view/about_section.R:
```diff
@@ -28,6 +28,31 @@ about_section <- shiny$div(
+  class = "about-section",
+  shiny$div(
+    shiny$markdown("
+> Monitor application adoption, track user engagement,
+and get usage logs for all your Shiny applications.
```
`>` looks odd

vituri replied:
Indeed! I removed the blockquote.

jakubnowicki commented on app/view/about_section.R:
```diff
@@ -28,6 +28,31 @@ about_section <- shiny$div(
 )
 
+about_section <- shiny$div(
+  class = "about-section",
```
The appearance of the new section is inconsistent with the overall design of the modal. Please adjust it - fix font, add "card-style" border, add section header.

vituri replied:
The blockquote was indeed odd, but I don't think that having no "card-style border" is an obvious _inconsistency_. The new section could be made of text, lists and a header, but need not to be inside a card. I had in mind the following example from other app in Appsilon's gallery. It uses a hr to separate the blocks.

![Screenshot from 2025-06-16 19-58-43](https://github.com/user-attachments/assets/b0522847-4990-4490-a343-b059fa7be132)

In our case we have bullets instead of accordions because our app is way simpler, but you get the idea.

In any case, I followed your suggestion.

jakubnowicki reviewed (changes requested):
The new section does not look good; please check my comments.

Please move the "Appsilon" part above the Rhino section. Powered by should be between Appsilon and Rhino.

Also, the modal itself looks bad on a larger screen:
![image](https://github.com/user-attachments/assets/76c82488-c6d6-487a-86d4-2e71b330e159)

Commit:
refine text of about_section; use same class as brand

Commit:
add link to brand section

Commit:
change order of blocks; add whitespace

Commit:
remove unused function

Commit:
refine text

Commit:
delete connect-extension

vituri commented:
## Changes:

1. I refined the text a bit
2. I used the same `brand-summary` / `brand-section` classes to the new section for consistency
3. Added a whitespace before the brand block
4. Added a "Learn more about Appsilon" link inside the brand block
5. Changed the order of brand and Rhino blocks

## Displaying

The modal in my screen:

![Screenshot from 2025-06-16 22-48-59](https://github.com/user-attachments/assets/43ffcbdb-04b6-4582-952c-bdb2c4ca9729)

The modal in a high resolution screen:

![Screenshot from 2025-06-16 22-49-13](https://github.com/user-attachments/assets/391aa2c2-46fc-45aa-ab0d-8ad8cb726cab)

The original modal:

![Screenshot from 2025-06-16 20-34-21](https://github.com/user-attachments/assets/3e486923-666e-430a-8895-1bf84ebdd157)

## Comparison

Two other info-modals I found on Appsilon Gallery, for comparison:

![Screenshot from 2025-06-16 20-02-05](https://github.com/user-attachments/assets/5a0ee450-40b7-4df8-85ca-75e7e47fe464)

![Screenshot from 2025-06-16 20-00-32](https://github.com/user-attachments/assets/4a2bd3a8-a810-4145-913d-9012359d3765)

## Questions

1. Should I keep the whitespace between the first and the second block?
2. Is the text ok? Should we add something like "Perfect for Connect admins, data teams" which gives an idea of the public who would use the app?
3. Should I shorten the section about Rhino, as you suggested earlier?

jakubnowicki committed:
chore: Fix modal styling.
